cmake_ext.py

```diff
@@ -7,18 +7,14 @@
 
 
 class CMakeExtension(Extension):
-    def __init__(self, sourcedir=""):
-        Extension.__init__(self, "CMakeExtension", sources=[])
```

Comment:
This was causing setuptools to look for `CMakeExtension.*.so`. This means this technique will not work if you produce multiple "Extensions" from a single "CMakeExtension" - but at some point we are really just recreating scikit-build from scratch.

Reply:
Agreed, this used to be simple enough to have a custom implementation, but not anymore.